presto-hive/src/test/java/io/prestosql/plugin/hive/AbstractTestHive.java

```diff
@@ -726,7 +733,7 @@ protected final void setup(String databaseName, HiveConfig hiveConfig, HiveMetas
 
         metastoreClient = hiveMetastore;
         HivePartitionManager partitionManager = new HivePartitionManager(hiveConfig);
-        hdfsEnvironment = HDFS_ENVIRONMENT;
+        hdfsEnvironment = new HdfsEnvironment(createTestHdfsConfiguration(), new HdfsConfig(), new NoHdfsAuthentication());
```

Comment:
HDFS_ENVIRONMENT is also used in the first `setup` method (the method calls the this one).

I'd suggest

1. move the new initialization of the `hdfsEnvironment` field to the first setup method
2. use `hdfsEnvironment` instead of `HDFS_ENVIRONMENT` everywhere